src/10-01.zig

```diff
@@ -6,24 +6,41 @@ pub fn main() !void {
     defer _ = gpa.deinit();
     const allocator = gpa.allocator();
 
+    // Deserialize JSON
     const json_str =
         \\{
         \\  "userid": 103609,
         \\  "verified": true,
         \\  "access_privileges": [
-        \\      "user",
-        \\      "admin"
+        \\    "user",
+        \\    "admin"
         \\  ]
         \\}
     ;
     const T = struct { userid: i32, verified: bool, access_privileges: [][]u8 };
     const parsed = try json.parseFromSlice(T, allocator, json_str, .{});
     defer parsed.deinit();
 
-    const value = parsed.value;
+    var value = parsed.value;
 
     std.debug.assert(value.userid == 103609);
     std.debug.assert(value.verified);
     std.debug.assert(std.mem.eql(u8, value.access_privileges[0], "user"));
     std.debug.assert(std.mem.eql(u8, value.access_privileges[1], "admin"));
+
+    // Serialize JSON
+    value.verified = false;
+    const new_json_str = try json.stringifyAlloc(allocator, value, .{ .whitespace = .indent_2 });
+    defer allocator.free(new_json_str);
+
+    std.debug.assert(std.mem.eql(u8, new_json_str,
```

Comment:
Thanks for contribution!

Maybe `std.testing.expectEqualStrings`  is better, Can you change it all together in this PR?